### PR TITLE
Fix: App-1837 - Setting Proposal Gas Estimation Error

### PR DESCRIPTION
**Pull request by WissenIstNacht**
## Description

Fixes the gas estimation error at the end of a settings proposal flow.

Task: [1837](https://aragonassociation.atlassian.net/browse/APP-1837)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by WissenIstNacht** [comment]
Aplogies, I first based my PR on the wrong branch. Now correctly based on the latest develop commit.

File: packages/web-app/src/pages/proposeSettings.tsx [hunk 1]
```diff
@@ -268,15 +284,20 @@ const ProposeSettingWrapper: React.FC<Props> = ({
   }, [daoToken, pluginSettings, getValues, setValue, tokenSupply?.raw]);
 
   useEffect(() => {
+    console.log('Is this even running?');
```
Comment (Fabricevladimir) [inline packages/web-app/src/pages/proposeSettings.tsx]: Why are you asking me things? :D

File: packages/web-app/src/pages/proposeSettings.tsx [hunk 2]
```diff
@@ -268,15 +284,20 @@ const ProposeSettingWrapper: React.FC<Props> = ({
...
       const actions: Array<Promise<DaoAction>> = [];
       if (!pluginClient || !client) return Promise.all(actions);
 
       for (const action of getValues('actions') as Array<Action>) {
+        console.log('Current action: ');
+        console.log('[LOGGING] action ' + JSON.stringify(action, null, 2));
         if (action.name === 'modify_metadata') {
           const ipfsUri = await client.methods.pinMetadata(action.inputs);
+          console.log('[LOGGING] dao ' + dao);
+          console.log('[LOGGING] ipfsUri ' + ipfsUri);
```
Comment (Fabricevladimir) [inline packages/web-app/src/pages/proposeSettings.tsx]: Le festival des logs, I see you! Btw, there are a couple I didn't catch last time in editSettings -> majorityVoting.tsx

**Review by Fabricevladimir — APPROVED** [review]
Nice catch on the date thing! Good to merge once you remove the stray logs

**Pull request merged**